potodo/_github.py

```diff
@@ -5,12 +5,12 @@
 from subprocess import check_output
 
 
-def get_repo_url() -> str:
+def get_repo_url(repo_path: str) -> str:
     """Tries to get the repository url from git commands
     """
-    url = check_output("git remote get-url --all upstream", universal_newlines=True)
+    url = check_output("git remote get-url --all upstream", universal_newlines=True, shell=True, cwd=repo_path)
```

Comment:
I'd restrict "shell" to where it's really needed like when we use shell redirections. But here it's not needed, I'd prefer a `.split()`, it's way faster.